### PR TITLE
Fix trimmed quantity inputs in draft order

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it fixes trimmed quantity inputs in draft order.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2751 Draft order lines display trimmed quantity](https://app.clickup.com/t/2549495/SALEOR-2751)

**Review by jwm0 — APPROVED** [review]

**Review by kamilpastuszka — COMMENTED** [review]
Wouldn’t it be better to try to make that quantity field responsive based on user input?
Now, there will be still issues with higher values.

**Comment by orzechdev** [comment]
We don't have responsive inputs in the dashboard, but you are right. I've changed it to resize input responsively based on available table size.

File: src/orders/components/OrderDraftDetailsProducts/TableLineForm.tsx [hunk 1]
```diff
@@ -13,7 +13,9 @@ const useStyles = makeStyles(
         padding: "12px 12px 10px",
         textAlign: "right"
       },
-      width: 60
+      width: "auto",
+      minWidth: 100,
+      maxWidth: 140
```
[the remark this answers is not in the document]
Comment (kamilpastuszka) [reply src/orders/components/OrderDraftDetailsProducts/TableLineForm.tsx:18]: I think 6-digit quantity would be enough in this case.

**Review by kamilpastuszka — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**